tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -2440,16 +2440,143 @@ TEST_F(EnergyPlusFixture, VRF_FluidTCtrl_CalcVRFIUTeTc)
+    EXPECT_LT(VRF(IndexVRFCondenser).IUEvaporatingTemp, 15);
+    // default value, coil inlet temps lower than default
+    EXPECT_EQ(VRF(IndexVRFCondenser).IUCondensingTemp, 42);
+    Real64 saveCoolingEvapTemp = VRF(IndexVRFCondenser).IUEvaporatingTemp;
+
+    // test blow thru fan
+    // adjust coil inlet temps for fan heat
+    VRFTU(1).coilInNodeT = Node(3).Temp;
+    VRFTU(1).coilInNodeW = Node(3).HumRat;
+    VRFTU(2).coilInNodeT = Node(3).Temp;
+    VRFTU(2).coilInNodeW = Node(3).HumRat;
+    VRFTU(1).FanPlace = DataHVACGlobals::BlowThru;
+    VRFTU(2).FanPlace = DataHVACGlobals::BlowThru;
+    // system is on in cooling mode
+    VRF(IndexVRFCondenser).CalcVRFIUTeTc_FluidTCtrl();
+    // fan heat added to coil inlet temp, coil surface temp should also be lower to overcome additional heat tranfer
+    EXPECT_LT(VRF(IndexVRFCondenser).IUEvaporatingTemp, saveCoolingEvapTemp);
+    // default value, coil inlet temps lower than default
+    EXPECT_EQ(VRF(IndexVRFCondenser).IUCondensingTemp, 42);
+
+    VRFTU(1).FanPlace = DataHVACGlobals::DrawThru;
+    VRFTU(2).FanPlace = DataHVACGlobals::DrawThru;
+    // rest coil inlet temps
+    VRFTU(1).coilInNodeT = Node(1).Temp;
+    VRFTU(1).coilInNodeW = Node(1).HumRat;
+    VRFTU(2).coilInNodeT = Node(2).Temp;
+    VRFTU(2).coilInNodeW = Node(2).HumRat;
+    CoolingLoad(1) = false;
+    HeatingLoad(1) = true;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).OutputRequiredToCoolingSP = 300.0;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).OutputRequiredToHeatingSP = 200.0;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(2).OutputRequiredToCoolingSP = 2000.0;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(2).OutputRequiredToHeatingSP = 1900.0;
+    // system is on in heating mode
+    VRF(IndexVRFCondenser).CalcVRFIUTeTc_FluidTCtrl();
+    // default value, coil inlet temps higher than default
+    EXPECT_EQ(VRF(IndexVRFCondenser).IUEvaporatingTemp, 15);
+    // maximum coil surface temp of both heating coils above default
+    EXPECT_GT(VRF(IndexVRFCondenser).IUCondensingTemp, 42);
+    Real64 saveHeatingCondTemp = VRF(IndexVRFCondenser).IUCondensingTemp;
+
+    // test blow thru fan
+    VRFTU(1).FanPlace = DataHVACGlobals::BlowThru;
+    VRFTU(2).FanPlace = DataHVACGlobals::BlowThru;
+    // adjust coil inlet temps for fan heat
+    VRFTU(1).coilInNodeT = Node(3).Temp;
+    VRFTU(1).coilInNodeW = Node(3).HumRat;
+    VRFTU(2).coilInNodeT = Node(3).Temp;
+    VRFTU(2).coilInNodeW = Node(3).HumRat;
+    // system is on in heating mode
+    VRF(IndexVRFCondenser).CalcVRFIUTeTc_FluidTCtrl();
+    // default value, coil inlet temps higher than default
+    EXPECT_EQ(VRF(IndexVRFCondenser).IUEvaporatingTemp, 15);
+    // fan heat added to coil inlet temp, coil surface temp should also be lower to account for less heat tranfer
+    EXPECT_LT(VRF(IndexVRFCondenser).IUCondensingTemp, saveHeatingCondTemp);
+
     // Clean up
     VRF.deallocate();
```

Comment:
Ideally these are cleaned up as part of the clear_state after every single unit test is run, but I understand it doesn't hurt to have them here.